prov/rxm/src/rxm_ep.c

```diff
@@ -451,7 +450,6 @@ static ssize_t rxm_ep_send_common(struct fid_ep *ep_fid, const struct iovec *iov
 	ret = rxm_get_conn(rxm_ep, dest_addr, &rxm_conn);
 	if (ret)
 		return ret;
-
 	if (freestack_isempty(rxm_ep->txe_fs)) {
 		FI_DBG(&rxm_prov, FI_LOG_CQ, "Exhaused tx_entry freestack\n");
 		return -FI_ENOMEM;
```

Comment:
I'll drop all changes related to RXM provider, it's just minor changes. All changes that are carry value already merged in master.